src/components/layout/ExtendLease/ExtendLease.tsx

```diff
@@ -159,7 +159,7 @@ function ExtendLease() {
           className="flex flex-row center"
           style={{ justifyContent: 'space-between' }}
         >
-          <h1 className="white">Extend Lease</h1>
+          <h1 className="white text-2xl">Extend Lease</h1>
```

Comment:
Not sure it's intentional, but this is smaller than what was previously in prod. (Renders as 24px size here). It's odd as 2xl should be 32px with a 16px base font size; unsure if the h1 is interfering somehow.

Reply:
Ah, it is 24px for 2xl. To get 32px use text-[2rem]